docs/content/guides/cell-features/merge-cells/merge-cells.md

```diff
@@ -66,6 +66,58 @@ To initialize Handsontable with predefined merged cells, provide merged cells de
 
 :::
 
+## Optimizing rendering of the wide/tall merged cells
+
+Sometimes, you might need to render a table with merged cells that are very wide or very tall. In such cases, the standard rendering mode may not be optimal. In these situations, we suggest enabling the virtual DOM for the merged cells.
```

Comment:
Maybe: When cells span thousands of rows or columns, scrolling may feel slower compared to unmerged cells. To improve performance, consider enabling the dedicated virtualization feature for merged cells, which is disabled by default.

Reply:
The current text (after @jansiegel review) appeals to me more than what you suggested. I'd leave it as it is.